packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -198,7 +211,7 @@ const shouldBundleModules = (projectStructure: ProjectStructure) => {
  * @param modulePath
  * @returns name of module if set by user via ModuleConfig
  */
-const getModuleName = (modulePath: string): string | undefined => {
+export const getModuleName = (modulePath: string): string | undefined => {
```

Comment:
If this is used by the plugin and dev then it should be moved to a common folder

Reply:
This wasn't moved

Reply:
Fixed